docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -238,7 +238,7 @@ In addition, there are several related configuration options that may need adjus
 ### Limitations
 
 - **Batch jobs only**: Adaptive Batch Scheduler only supports batch jobs. Exception will be thrown if a streaming job is submitted.
-- **BLOCKING or HYBRID jobs only**: At the moment, Adaptive Batch Scheduler only supports jobs whose [shuffle mode]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) is `ALL_EXCHANGES_BLOCKING / ALL_EXCHANGES_HYBRID_FULL / ALL_EXCHANGES_HYBRID_SELECTIVE`.
+- **BLOCKING or HYBRID jobs only**: At the moment, Adaptive Batch Scheduler only supports DataStream jobs whose [shuffle mode]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) is `ALL_EXCHANGES_BLOCKING / ALL_EXCHANGES_HYBRID_FULL / ALL_EXCHANGES_HYBRID_SELECTIVE` and DataSet jobs whose `ExecutionMode` is `BATCH_FORCED`.
```

Comment:
I would prefer to clarify here on how to enforce the ALL BLOCKING for DataSet jobs, considering that the usage of AdaptiveBatch with DataSet jobs has already been detailed in the official documentation for users.

Reply:
I think we can not directly use `DataStream jobs` here, because it does not include SQL/Table jobs.

Reply:
How about `DataStream API` and `DataSet API`, which are used in many FLIP content ?

Reply:
I personlly think that `DataStream API` does not include `SQL/Table API`

Reply:
Maybe keep the original statement and add an extra statement like "Note that for DataSet jobs, the `ExecutionMode` needs to be `BATCH_FORCED` since they do not use the aforementioned shuffle mode".

Later in Flink 2.0, we can directly remove the extra statement.